lib/broccoli/ember-app.js

```diff
@@ -1534,6 +1534,8 @@ EmberApp.prototype.toTree = function(additionalTrees) {
  */
 EmberApp.prototype.contentFor = function(config, match, type) {
   var content = [];
+  var deprecatedHooks = ['app-prefix', 'app-suffix'];
+  var deprecate = this.project.ui.writeDeprecateLine.bind(this.project.ui);
```

Comment:
That could probably be better documented. I built it so it to mirror the `Ember.warn` (a falsy test causes the message to be logged).